### PR TITLE
Changelog for 0.6.0

### PR DESCRIPTION
**Pull request by rgrunber**
Signed-off-by: Roland Grunberg <rgrunber@redhat.com>

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,41 @@
 # Tools for MicroProfile Changelog
 
+## [0.6.0](https://github.com/redhat-developer/vscode-microprofile/milestone/6?closed=1) (November 29, 2022)
+
+### Enhancements
```
Comment (datho7561) [inline CHANGELOG.md:5]: I just merged the "activation trigger should be more selective" PR, I think that should be listed here

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,41 @@
...
+
+### Bug Fixes
+ * Don't throw an error popup when MicroProfile LS cancels the inlay hint process. See [#123](https://github.com/redhat-developer/vscode-microprofile/pull/123).
+ * Java source code not validated upon start. See [eclipse/lsp4mp#301](https://github.com/eclipse/lsp4mp/issues/301).
+ * `ClassCastException` thrown (and caught) when using invalid `@ConfigProperty` default value. See [eclipse/lsp4mp#295](https://github.com/eclipse/lsp4mp/issues/295).
+ * Improve handling of `@ConfigProperties` for validation. See [eclipse/lsp4mp#304](https://github.com/eclipse/lsp4mp/issues/304)
+ * Support for the `config_ordinal` property in `microprofile-config.properties`. See [eclipse/lsp4mp#289](https://github.com/eclipse/lsp4mp/issues/289).
+ * Property evaluation should support the environment variable default value notation. See [eclipse/lsp4mp#241](https://github.com/eclipse/lsp4mp/issues/241).
+ * Display property when hovering over a key that isn't defined in the application. See [eclipse/lsp4mp#285](https://github.com/eclipse/lsp4mp/issues/285).
```
Comment (datho7561) [inline CHANGELOG.md:19]: Display property value....

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,41 @@
 # Tools for MicroProfile Changelog
 
+## [0.6.0](https://github.com/redhat-developer/vscode-microprofile/milestone/6?closed=1) (November 29, 2022)
```
Comment (datho7561) [inline CHANGELOG.md:3]: Make sure to include any unique fixes from https://github.com/redhat-developer/vscode-microprofile/milestone/6?closed=1. I think there are a few there we should add

**Review by datho7561 — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 3 above)
Comment (rgrunber) [reply CHANGELOG.md:3]: I wasn't really sure about including this since, there isn't really supposed to be anything that is visible for the user.

Comment (datho7561) [reply CHANGELOG.md:3]: Okay. That makes sense!

**Review by datho7561 — APPROVED** [review]
Looks good. Thanks Roland!

**Comment by fbricon** [comment]
please add https://github.com/redhat-developer/vscode-microprofile/pull/131

**Pull request merged**